core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -269,7 +269,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("metrics directory")
       .ofType(classOf[java.lang.String])
-    val newConsumerOpt = parser.accepts("new-consumer", "Use the new consumer implementation. This is the default.")
+    val newConsumerOpt = parser.accepts("new-consumer", "DEPRECATED Use the new consumer implementation. This is the default.")
```

Comment:
Maybe at the end, "This is the default, so this option is deprecated and will be removed in a future release." Otherwise it may seem like using the new consumer is deprecated.